ui.go

```diff
@@ -35,10 +35,56 @@ func (pg *ProcessListGrid) SetText(str string) {
 }
 
 func (pg *ProcessListGrid) OnResize(payload ui.Resize) {
-	pg.grid.SetRect(0, 10, payload.Width, payload.Height-10)
+	pg.grid.SetRect(0, 15, payload.Width, payload.Height-15)
 	ui.Render(pg.grid)
 }
 
 func (pg *ProcessListGrid) Render() {
 	ui.Render(pg.grid)
 }
+
+type HotSpotGrids struct {
+	grids []*ui.Grid
+}
+
+func newHotSpotGrids() *HotSpotGrids {
+	termWidth, _ := ui.TerminalDimensions()
+	ret := &HotSpotGrids{}
+	offset := 0
+	for i := 0; i < 5; i++ {
```

Comment:
Could you explain the magic number 5?

Reply:
I want to show top10 hot regions, which means we need 5 rows with 2 columns.